package.json

```diff
@@ -27,10 +27,8 @@
     "setup": "corepack enable pnpm",
     "build": "tsup src/index.ts --format esm,cjs --dts",
     "test": "vitest",
-    "lint": "biome check .",
-    "format": "pnpm format:biome:format && pnpm format:biome:check",
-    "format:biome:format": "biome format --write .",
-    "format:biome:check": "biome check --apply ."
+    "lint": "biome lint .",
```

Comment:
From the user's point of view, I think `biome lint` is not sufficient since it does not check `format files` and `organize imports`.
I think it would be better to leave `biome check` as it is. What do you think?